### ISSUE TITLE
Maximized window or Fullscreen mode [FS]

### ISSUE THREAD

**Issue by PatchedInfinities** [DaIsimsiz/planetDefs-Builder#1]
**Is your feature request related to a problem? Please describe.**
Not that I'm aware. It just feels nice being able to maximize a window.

**Describe the solution you'd like**
Maximizing the window with the square on the top bar (no idea what that bar is called, just took a screenshot) or going fullscreen with f11

**Describe alternatives you've considered**
None, I don't even know basic C#, so I can't be of much help fixing it myself.

**Additional context**
This square:
![image](https://github.com/DaIsimsiz/planetDefs-Builder/assets/165608774/14aa485e-ee9d-40b8-9cf9-a8c5061e6406)

**Comment by DaIsimsiz** [comment]
I limited the app size to a specific size to not break my layout code.
Maybe if I rewrite the app later I may unlock resizing, but it will stay locked for now, sorry!

TL;DR It's intended
